docs/configuration.rst

```diff
@@ -36,20 +36,40 @@ for more information on how each engine can be configured.
 If your job callback is performing a database involved operation, you should check if the CakePHP database connection is still alive.
 It can happen that your database connection timed out, when no jobs were acknowledged for a long time.
 A way to come around this, is to disconnect the CakePHP connection every time a job succeeded or failed.
+
 To do so, create a subclass of the ``QueuesadillaShell`` and implement two event listeners inside the ``getWorker`` method.
+You can easily do that by baking a shell with ``bin/cake bake shell MyQueuesadillaShell``  command and alter the created
+class as below:
 
 .. code:: php
+    <?php
+    namespace App\Shell;
+
+    use Cake\Datasource\ConnectionManager;
+    use Josegonzalez\CakeQueuesadilla\Shell\QueuesadillaShell;
 
-    public function getWorker($engine, $logger)
+    /**
+     * AlteredQueuesadillaShell shell command.
+     */
+    class AlteredQueuesadillaShellShell extends QueuesadillaShell
```

Comment:
ShellShell? Seems like maybe when you bake, no need to specify the suffix `Shell`?